package/MDAnalysis/analysis/density.py

```diff
@@ -1264,6 +1264,9 @@ def Bfactor2RMSF(B):
     ----------
 
     .. [Willis1975]  BTM Willis and AW Pryor. *Thermal vibrations in crystallography*. Cambridge Univ. Press, 1975
+
+
+    .. deprecated:: 1.0.0
```

Comment:
Here you will need to provide more details for the deprecation notice. See the deprecation notice underneath `density_from_PDB` for an idea of what to write (possibly mention that it's going away as part of the scheduled removal of `density_from_PDB`)?